storage/innobase/buf/buf0buf.cc

```diff
@@ -495,6 +495,9 @@ buf_get_total_stat(
 
 		tot_stat->n_pages_not_made_young +=
 			buf_stat->n_pages_not_made_young;
+
+		tot_stat->buf_lru_flush_page_count +=
+		    buf_stat->buf_lru_flush_page_count;
```

Comment:
What about other 2 remaining data members
```
	ulint	LRU_bytes;	/*!< LRU size in bytes. Protected by
				LRU_list_mutex. */
	ulint	flush_list_bytes;/*!< flush_list size in bytes.
				Protected by flush_list_mutex */
```
https://github.com/percona/percona-server/blob/eac6473879d484f6df72240928d964dc562c8842/storage/innobase/include/buf0buf.h#L2089
and
https://github.com/percona/percona-server/blob/eac6473879d484f6df72240928d964dc562c8842/storage/innobase/include/buf0buf.h#L2091

Reply:
Thanks or looking @percona-ysorokin.  I thought the same thing, but it seems that these are not actually exposed in SHOW STATUS, so I left them as they are.